simplesat/constraints/tests/test_requirement.py

```diff
@@ -144,7 +144,18 @@ def test_any(self):
         self.assertTrue(requirement.matches(V("1.9.0-1")))
         self.assertEqual(requirement, r_requirement)
         self.assertEqual(requirement, r_requirement_empty)
+        self.assertEqual(r_requirement, r_requirement_empty)
+
+        # Given
+        requirement_string = "numpy *"
```

Comment:
they actually appear because we translate `Any()` into `*` when doing `str(requirement)`, so those come up anytime a package has a dependency in name only.

The alternative would be to remove `*` from `str(requirement)`